config.go

```diff
@@ -45,29 +51,33 @@ func (dc *KleanerConfig) SetFunctions(fnMap map[string]DeleteFunc) {
 }
 
 func init() {
+	v := viper.New()
```

Comment:
since I'm new to this project. What does viper do?

Reply:
[Viper](https://github.com/spf13/viper) is a project that basically allows for Go configuration.